CHANGELOG.md

```diff
@@ -61,6 +59,9 @@
 ### Jsonnet
 
 * [CHANGE] Memcached: Change default read timeout for chunks and index caches to `750ms` from `450ms`. #7778
+* [CHANGE] Fine-tuned `terminationGracePeriodSeconds` for the following components: #7364
```

Comment:
is this entry intentional?

Reply:
Yes, I moved it to the jsonnet section because it was a jsonnet change, not a mixin change.